test/amazonica/test/s3.clj

```diff
@@ -420,3 +417,8 @@
     "foo@"
     "foo@bar"
     "foo@bao.c"))
+
+(deftest no-internal-methods-exported-tests
+  (is (= nil
+         (resolve 'amazonica.aws.s3/access$000)
+         (resolve 'amazonica.aws.s3/access$200))))
```

Comment:
Note: I've verified that _this_ test passes, but the tests for the S3 test namespace fail.